### DISCUSSION TITLE
LOG_LEVEL does not work when running via terminal

### DISCUSSION THREAD

**Discussion by k2snowman69** [renovatebot/renovate#9995] [Request Help]
**What Renovate type, platform and version are you using?**

node/npm - "renovateVersion": "25.20.1"

**Renovate type**

- [ ] Hosted app
- [x] Self hosted

"renovateVersion": "25.20.1"

**Renovate platform**

- [ ] Azure DevOps (dev.azure.com)
- [ ] Azure DevOps Server (self-hosted)
- [ ] Bitbucket Cloud (bitbucket.org)
- [x] Bitbucket Server (self-hosted)
- [ ] Gitea (gitea.com, self-hosted)
- [ ] GitHub (github.com, GitHub Enterprise)
- [ ] GitLab (gitlab.com, self-hosted)

**Renovate version (if self hosting)**

I'm self hosting Renovate with version: 25.20.1

**Describe the bug**

Really hoping this is me being dumb... upgrading from renovate@24 to renovate@25 logLevel was removed from the config and meant to be set via the environment variables. So I wrote the following make file:
```
all: 
	RENOVATE_CONFIG_FILE=./config.js
	LOG_LEVEL=debug
	npx --no-install renovate
```

However when I run make I only see info level logs, not debug level logs:
```
 INFO: Repository started (repository=<redacted>)
       "renovateVersion": "25.20.1"
 INFO: Dependency extraction complete (repository=<redacted>)
       "baseBranch": "main",
       "stats": {
         "managers": {
           "dockerfile": {"fileCount": 1, "depCount": 2},
           "npm": {"fileCount": 1, "depCount": 23}
         },
         "total": {"fileCount": 2, "depCount": 25}
       }
 INFO: Repository finished (repository=<redacted>)
       "durationMs": 7159
```

I then tried running everything through the terminal directly:
```
$ RENOVATE_CONFIG_FILE=./config.js
$ LOG_LEVEL=debug
../node_modules/.bin/renovate
```

And still ended up with the same output

Fairly sure it's something dumb but looking through the code I didn't misspell [LOG_LEVEL](https://github.com/renovatebot/renovate/blob/main/lib/logger/index.ts#L19) so I figured I would ask

**Relevant debug logs**

<details><summary>Click me to see logs</summary>

```
 INFO: Repository started (repository=<redacted>)
       "renovateVersion": "25.20.1"
 INFO: Dependency extraction complete (repository=<redacted>)
       "baseBranch": "main",
       "stats": {
         "managers": {
           "dockerfile": {"fileCount": 1, "depCount": 2},
           "npm": {"fileCount": 1, "depCount": 23}
         },
         "total": {"fileCount": 2, "depCount": 25}
       }
 INFO: Repository finished (repository=<redacted>)
       "durationMs": 7159
```

</details>

**Have you created a minimal reproduction repository?**

Please read the [minimal reproductions documentation](https://github.com/renovatebot/renovate/blob/main/docs/development/minimal-reproductions.md) to learn how to make a good minimal reproduction repository.

- [x] I have provided a minimal reproduction repository
- [ ] I don't have time for that, but it happens in a public repository I have linked to
- [ ] I don't have time for that, and cannot share my private repository
- [ ] The nature of this bug means it's impossible to reproduce publicly

**Additional context**

The repository you run this on doesn't matter so fill ./config.js with whatever is easiest. This seems to be an environment variable issue.

**Comment by rarkins** [comment]
https://www.serverlab.ca/tutorials/linux/administration-linux/how-to-set-environment-variables-in-linux/

**Reply by rarkins** [reply]
`FOO=bar` disappears once you run it. You need `export FOO=bar` to persist it. But also, make files may be a different matter because they can reset state between lines. Easiest normally to put them on one line. e.g. `RENOVATE_CONFIG_FILE=./config.js LOG_LEVEL=debug ../node_modules/.bin/renovate`
